shashi/practiceTest2.js

```diff
@@ -0,0 +1,67 @@
+//calculate Factorial of a number
+const calcFactorial = num => {
+  if(num < 0){
+    return `Not a value number to calculate factorial`;
+  } 
```

Comment:
Don't return different data types, only null is acceptable. Yes, there are cases where you MIGHT return a different dataType, but for best practice don't. 

What data type should this function return?